database/changesets/20210422_ContinuousWorkloadTable.xml

```diff
@@ -14,14 +14,14 @@
                runInTransaction="false">
         <sql>
             CREATE TABLE ContinuousWorkload (
-                id              BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,          -- primary key
-                source_type     SOURCE NOT NULL,                                           -- type of this source
-                source_item     TEXT NOT NULL,                                             -- the identifier of the source table
-                executor_type   EXECUTOR NOT NULL,                                         -- type of this executor
-                executor_item   TEXT NOT NULL,                                             -- the identifier of the executor table
-                sink_type       SINK  NOT NULL,                                            -- type of this sink
-                sink_item       TEXT  NOT NULL,                                            -- the identifier of the sink table
-                updated         TIMESTAMPTZ DEFAULT (NOW() AT TIME ZONE 'utc') NOT NULL,   -- when was this workload updated
+                id              BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,        -- primary key
+                source_type     SOURCE,                                                  -- type of this source
+                source_items    TEXT NOT NULL,                                           -- the identifier of the source table
```

Comment:
I'm confused at the pluralization of item + the comment referencing a single identifier, can you clarify?